README.md

```diff
@@ -6,10 +6,39 @@ This is testing having a remote registry for [Singularity HPC](https://github.co
 
 📖️ Read the [documentation](https://singularity-hpc.readthedocs.io/en/latest/) 📖️
 ⭐️ Browse the [container module collection](https://singularityhub.github.io/shpc-registry/) ⭐️
- 
+
+## Contribution rules
+
+This registry is there to showcase Singularity HPC, and provide a workable default set of containers.
+Contributions are very much welcome, so please do submit a pull-request if you'd like more software to be added !
+
+The documentation has an [introduction](https://singularity-hpc.readthedocs.io/en/latest/getting_started/developer-guide.html#writing-registry-entries) about registries, and explain how to organise the various files.
+
+### Automated generation
+
+First of all, there is an automated way of getting most of the `container.yaml` written via GitHub magic 🎩 automation.
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this repository.
```

Comment:
Do they need to fork the repository? E.g., should we make an issue template that requests a container, and then we can easily trigger (I'm happy to do this). So maybe we could suggest to open an issue to request a container URI, and (optionally) for advanced users they can fork, run the workflow, and PR.

Reply:
How would you handle editing the list of aliases ? By default, people wouldn't be allowed to edit the PR. Unless you change the permissions for that branch ?

Reply:
Hmm that's a good point - do you want to try it out and see what we can do?

Reply:
I've added an issue form 7692c0dcf3214f3adda6fe34aa23d9d5aec872e7 to collect the requirements